woof-code/rootfs-packages/acpid_busybox/etc/acpi.map

```diff
@@ -0,0 +1,4 @@
+"EV_KEY"	0x01	"KEY_POWER"	116	1	"button/power PWRF 00000080"
+"EV_KEY"	0x01	"KEY_POWER"	116	1	"button/power PWRB 00000080"
+"EV_SW"		0x05	"SW_LID"	0	1	"button/lid LID0 00000080"
```

Comment:
The first 3 are hardcoded in busybox acpid.c, if acpi.map is missing - by default, it runs /etc/acpi/LID/00000080 only on lid close